target/linux/ramips/image/mt7621.mk

```diff
@@ -213,6 +262,38 @@ define Device/asus_rt-n56u-b1
 endef
 TARGET_DEVICES += asus_rt-n56u-b1
 
+define Device/beeline_smartbox-pro
+  $(Device/dsa-migration)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  UBINIZE_OPTS := -E 5
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 30720k
+  KERNEL_LOADADDR := 0x81001000
+  LZMA_TEXT_START := 0x82800000
+  KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | \
+    lzma | uImage lzma
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | loader-kernel | \
+    lzma | uImage lzma
+  LOADER_TYPE := bin
+  IMAGES += kernel.bin rootfs.bin factory.img
+  IMAGE/kernel.bin := append-ubi | sercomm-tag-header-kernel 0x1700100 0x1f00000
+  IMAGE/rootfs.bin := append-ubi | check-size
+  IMAGE/factory.img := append-ubi | sercomm-tag-factory-img-pro 0x1700100 0x1f00000 0x1b00100 0x3d00000
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
```

Comment:
What do we need all these images for? Commit message only mentions sysupgrade and initramfs.

Reply:
Left for the duration of the tests. Then leave only factory & sysupgrade?